pyproject.toml

```diff
@@ -26,8 +26,8 @@ dependencies = [
 	"resampy>=0.2.2,<0.4.3",
 	"scikit-learn",
 	"scipy>=1.4.1",
-	"tensorflow>=2.4.1,<2.16; platform_system != 'Darwin' and python_version >= '3.11'",
-	"tensorflow-macos>=2.4.1; platform_system == 'Darwin' and python_version >= '3.11'",
+	"tensorflow>=2.4.1,<2.15.1; platform_system != 'Darwin' and python_version >= '3.11'",
```

Comment:
Why wouldn't Tensorflow 2.15.1 be supported?

Reply:
We can't load our keras model in 2.15.1 yet. Unsure why. 2.16.1 also behaves similarly.